src/test/run-make-fulldeps/output-type-permutations/Makefile

```diff
@@ -78,7 +78,7 @@ all:
 	rm $(TMPDIR)/$(call BIN,foo)
 	$(RUSTC) foo.rs --crate-type=dylib --emit=link=$(TMPDIR)/$(call BIN,foo)
 	rm $(TMPDIR)/$(call BIN,foo)
-	rm -f $(TMPDIR)/{lib,}foo.{dll.exp,dll.lib,pdb,dll.a,exe.lib}
```

Comment:
Shouldn't `lib` be kept here for the MSVC target?

Reply:
I think this test is just wrong, it tells `rustc` to create DLL with `.exe` suffix. LD will create importlib for it since https://github.com/mati865/rust/commit/5af318bd563345c0205dfd1060e90c0368054dc6#diff-097f3c6ebc9d98f0cc511c6dec798584, MSVC should be unaffected.